src/utils/getAccessTabData.ts

```diff
@@ -55,12 +70,36 @@ const getResourceLocators = (searchItem: Record<string, any>): string => {
   if (resourceLocatorString === '') {
     const { organisationValue, email } = getOrganisationDetails(searchItem?._source, true);
     if (organisationValue && email) {
-      resourceLocatorString = `For access contact ${organisationValue} :- ${email}`;
+      resourceLocatorString = `Contact organisation for Resource locator information`;
     }
   }
   return resourceLocatorString;
 };
 
+const getContactInformation = (searchItem: Record<string, any>): string => {
+  const contactInformationArray: string[] = [];
+
+  const sortedArrayAndCombinedContact = combineAndSortContacts(
+    searchItem?._source?.contact,
+    searchItem?._source?.contactForResource,
+  );
+
+  if (sortedArrayAndCombinedContact.length > 0) {
+    sortedArrayAndCombinedContact.forEach((contact) => {
+      const contactInfo = contact.email
```

Comment:
as contact is of type any. either define the interface or do a safe reading here

Reply:
Resolved by adding interface